scripts/Muon/GUI/Common/plotting_widget/plotting_widget_view.py

```diff
@@ -138,19 +165,65 @@ def get_selected(self):
         return self.plot_selector.currentText()
 
     def on_plot_button_clicked(self, slot):
-        self.plot_button.clicked.connect(slot)
+        pass
```

Comment:
If the plot button has been removed do we need this function at all anymore?

Reply:
I was unsure at the time as to whether we needed it, I've now fully removed it.